package.json

```
@@ -121,42 +121,45 @@
     "test": "aegir lint",
     "build": "aegir build",
     "start": "node dist/src/index.js",
-    "release": "npm run build && aegir release --test false --docs false"
+    "dev-start": "NODE_OPTIONS='--unhandled-rejections=strict --trace-atomics-wait --trace-deprecation --trace-exit --trace-sigint --trace-uncaught --trace-warnings' ts-node-esm --esm -T --preferTsExts --logError src/index.ts",
+    "release": "npm run build && aegir release --test false --docs false",
```

Aegir doesn't run tests or generate docs on release any more.

```suggestion
    "release": "npm run build && aegir release",
```